libs/pgwire/src/main/antlr/PgArray.g4

```diff
@@ -70,14 +65,4 @@ fragment HEX
     : [0-9a-fA-F]
     ;
 
-fragment DIGIT
-    : '0' | [1-9] [0-9]*
-    ;
-
-fragment EXP
-    : [Ee] [+\-]? DIGIT
-    ;
-
-WS
-   : [ \t\n\r] + -> skip
-   ;
+WS: [ \t\n\r]+ -> skip;
```

Comment:
Is this still used?

Reply:
yes it is required to drop the trailing/leading whitespaces around items or before/after the initial '{' or closing '}'